resources/validations/src/CIS.xsl

```diff
@@ -0,0 +1,463 @@
+                    <style><xsl:value-of disable-output-escaping="true" select="replace($css, '\s+', ' ')" /></style>
+                </xsl:if>
+            </head>
+            <body>
+                <h1>
+                    <xsl:value-of
+                        select="$document-title" />
+                </h1>
+                <p>
+                    <xsl:text expand-text="true">This summary was produced { format-dateTime(current-dateTime(), '[MNn] [D] [Y] [H01]:[m01] [ZN,*-3]') }.</xsl:text>
+                </p>
+                <p>
+                    <xsl:text>The system security plan analyzed is </xsl:text>
+                    <cite>
+                        <xsl:value-of
+                            select="$ssp//metadata/title" />
+                    </cite>
+                    <xsl:text> last modified </xsl:text>
+                    <xsl:value-of
+                        select="format-dateTime($ssp//metadata/last-modified, '[MNn] [D] [Y]')" />
+                    <xsl:text>.</xsl:text>
+                </p>
+                <p>The <code>security-sensitivity-level</code> of this system security plan is <code><xsl:value-of
+                            select="$ssp//security-sensitivity-level" /></code>.</p>
+                <p>
+                    <xsl:text>The FedRAMP reference catalog is </xsl:text>
+                    <a
+                        target="_blank"
+                        href="{$catalog-url} ">
+                        <cite
+                            class="url"
+                            title="{$catalog-url}">
+                            <xsl:value-of
+                                select="$catalog//metadata/title" />
+                        </cite>
+                    </a>
+                    <xsl:text>: version </xsl:text>
+                    <code
+                        class="version">
+                        <xsl:value-of
+                            select="$catalog//metadata/version" />
+                    </code>
+                    <xsl:text> last modified </xsl:text>
+                    <xsl:value-of
+                        select="format-dateTime($catalog//metadata/last-modified, '[MNn] [D] [Y]')" />
+                    <xsl:text>.</xsl:text>
+                </p>
+                <p>There are <xsl:value-of
+                        select="count($catalog//control)" /> required controls at that security-sensitivity-level having a total of <xsl:value-of
+                        select="
+                            count($catalog//prop[@name = 'response-point']/parent::part[matches(@id, 'smt')])" /> response
+                    points.</p>
+
+                <xsl:call-template
+                    name="nested" />
+
+            </body>
+
+        </html>
+    </xsl:template>
+
+    <xsl:template
+        name="nested">
+        <h2>Nested Control Implementation Summary</h2>
+        <details>
+            <summary>Legend</summary>
```

Comment:
B: I think we should discuss Legend Symbols from a UX and UI approach. I am not saying I dislike them, but we definitely ought to align that with the rest of the user interface.

Reply:
I agree. Our semiotician will guide us in this endeavor.